cluster/examples/kubernetes/nfs/ceph-pvc.yaml

```diff
@@ -0,0 +1,13 @@
+# A rook ceph cluster must be running
+# Create a rook ceph cluster using examples in rook/cluster/examples/kubernetes/ceph
```

Comment:
maybe link to the quick start docs also? https://rook.io/docs/rook/master/quickstart.html